e2e/cypress/tests/integration/archived_channel/archived_leave_channel_spec.ts

```diff
@@ -40,7 +40,7 @@ describe('Leave an archived channel', () => {
     });
 
     it('MM-T1670 Can view channel info for an archived channel', () => {
-        cy.apiCreateArchivedChannel('archived-a', 'Archived A', 'O', testTeam.id).then((channel) => {
+        cy.apiCreateArchivedChannel('archived-a', 'Archived A', 'O', testTeam.id, [], null).then(({channel}) => {
```

Comment:
The return value here is not correct. Could you please correct the one type definition for `apiCreateArchivedChannel`? Also, I suggest for the last. two parameter to be optional so we don't need to pass `[], null`.

Reply:
Fixed in b1cbaae6cf563bbeb3070efb22f97a05c8787855